dxp/react_rendering.md

```diff
@@ -0,0 +1,148 @@
+# React Rendering in DXP
+
+This article describes DXP infrastructure to render React components from the server side perspective. This includes taglibs as well as low level services.
```

Comment:
"low level" → "low-level"

Reply:
What is the rule? When do you use `-` or nothing? Is there a rule, in fact?

Reply:
I will probably do a bad job of explaining this, but when you have a `modifier` + `noun` which act together as though they were a single "atomic" `adjective`, you join them with a hyphen.

Looks like the official term for these is "compound adjectives":

Probably a better explanation: https://www.gingersoftware.com/content/grammar-rules/adjectives/compound-adjectives/

Reply:
:-o Thanks :+1: